0003-use-of-single-signon.md

```diff
@@ -0,0 +1,78 @@
+# Provisions for the use of single sign on when accessing treetracker web applications
+
+* Status: proposed 
+<!-- {proposed | rejected | accepted | deprecated | … | superseded by [ADR-0005](0005-example.md)} --> <!-- optional -->
+* Deciders: @ZavenArra @Dadiorchen @nmcharlton @gwynndp
+<!-- Find deciders here: https://github.com/orgs/Greenstand/people  -->
+* Date: 2022-05-11
+
+Technical Story: 
+* https://greenstand.gitbook.io/auth-system/
+* https://github.com/orgs/Greenstand/projects/47
+
+## Context and Problem Statement
+
+Keycloak is being implemented to replace the current auth system used by the admin panel, and to provide a more generalized auth system for web apps and microservices.  Keycloak offers single sign on (SSO) integrations for accounts from other providers, most notably Google and Facebook.  We should decide which, if any, SSO providers to allow our users to use when accessing our systems.
+
+## Decision Drivers <!-- optional -->
+
+* Ease of use for our users
+* Ease of implementation
+* Exposing our users to negative consequences from other providers
+* Implicit support of other providers technology philosophy
+
+## Considered Options
+
+1. Do not implement SSO
```

Comment:
Can you explain a little more about this, you mean, don't use Keycloak at all or use Keycloak but do not support 3rd part provider, which means use the built-in user account provider in Keycloak?